docs/headless-cms/docs/03-01-headless-cms-in-console.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Headless CMS in Console
+type: Details
+---
+
+Headless CMS in Kyma is a separate component and does not provide templates or a standalone UI to display content from CMS. Still, some of its assets are shown in the Console UI in the Docs view (generic and component-related documentation) and the Service Catalog views (API specifications).
+
+You add a new documentation topic to the Console UI by creating a CR, while the [Asset Store](/components/asset-store/#overview-overview) handles the rest. You define the location, grouping, and order of the documents through DocsTopic and ClusterDocsTopic labels which you add to the custom resource definition. You can create your own labels and follow your own naming convention, but **cms.kyma-project.io/{label-name}** applies to the Console UI.
```

Comment:
```suggestion
You simply add a new documentation topic to the Console UI by creating a CR, while the [Asset Store](/components/asset-store/#overview-overview) handles the rest. You can also define the location, grouping, and order of the documents through DocsTopic and ClusterDocsTopic labels which you add to the custom resource definition. You can create your own labels and follow your own naming convention, but **cms.kyma-project.io/{label-name}** applies to the Console UI.
```
I don't know why but `you add` and `you define` sounds odd here :S hence the additional words, but I may be wrong :)

Reply:
you are right ;)